specification/marketplacecatalog/data-plane/Reviews/preview/2023-01-01-preview/reviews.json

```
@@ -0,0 +1,355 @@
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 - In case request malformed.\n\n * 500 - In case internal error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/products/{uniqueProductId}/reviews/{reviewId}/comments": {
+      "get": {
+        "tags": [
+          "Reviews Public API"
+        ],
+        "summary": "Get the comments of a given review",
+        "operationId": "ReviewComment_GetComments",
+        "x-ms-examples": {
+          "Facets_Get": {
+            "$ref": "./examples/GetReviewComments.json"
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "type": "string",
+            "name": "uniqueProductId",
+            "in": "path",
+            "required": true,
+            "description": "Unique Product Id"
+          },
+          {
+            "type": "string",
+            "name": "reviewId",
+            "in": "path",
+            "required": true,
+            "description": "Review Id"
+          },
+          {
+            "type": "integer",
+            "name": "maxpagesize",
+            "in": "query",
+            "description": "maximum number of comments to return",
+            "format": "int32",
+            "default": 50
+          },
+          {
+            "type": "string",
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "description": "Version of the REST API used for the request."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns a list of comments for the given review",
+            "schema": {
+              "$ref": "#/definitions/PageResultOfReviewCommentDto"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 - In case request malformed.\n\n * 500 - In case internal error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ErrorResponse": {
+      "description": "The Rating and Reviews error response object",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The search error response details",
+          "$ref": "#/definitions/ErrorResponseDetails"
+        }
+      }
+    },
+    "ErrorResponseDetails": {
+      "description": "The Rating and Reviews error response details object",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The error code description. Such as code=InternalError",
+          "type": "string"
+        },
+        "message": {
+          "description": "The error message. Such as message=Internal server error",
+          "type": "string"
+        }
+      }
```

The "code" and "message" properties should be required.
```suggestion
      },
      "required": ["code", "message"]
```

---

Done